src/AspNetCore.Identity.MongoDB/project.json

```diff
@@ -10,9 +10,9 @@
         "tags": ["identity", "mongodb", "mongo", "aspnetcore", "auth"]
     },
     "dependencies": {
-        "Microsoft.AspNetCore.Identity": "1.0.0",
-        "Microsoft.Extensions.Logging.Abstractions": "1.0.0",
-        "MongoDB.Driver": "2.3.0"
+        "Microsoft.AspNetCore.Identity": "1.1.0",
+        "Microsoft.Extensions.Logging.Abstractions":"1.1.0",
+        "MongoDB.Driver": "2.4.1"
```

Comment:
Also, we should rely on `Microsoft.AspNetCore.Identity` v1.0.0 I think as I would like to support that, too.